kie-api/src/build/revapi-config.json

```diff
@@ -35,7 +35,12 @@
         "code" : "java.method.addedToInterface",
         "new" : "method org.kie.api.builder.model.KieBaseModel org.kie.api.builder.model.KieModuleModel::newKieBaseModel()",
         "justification" : "Makes KieBase name optional."
+      },
+      {
+        "code" : "java.method.removed",
+        "old" : "method void org.kie.api.runtime.process.EventListener::signalEvent(java.lang.String, java.lang.Object)",
+        "justification" : "The .signalEvent(String, Object) method has been moved to the (parent) EventSignallable interface."
```

Comment:
Not directly related, but should revapi report this as a API breakage? If the method is in parent interface, the end result is the same?

Just trying to see if this is a bug in revapi in which case we could report it...